packages/launchpad/cypress/e2e/global-mode.cy.ts

```diff
@@ -30,8 +30,11 @@ describe('Launchpad: Global Mode', () => {
         .dropFileWithPath(projectPath)
       })
 
-      cy.contains('Welcome to Cypress!')
-      cy.get('a').contains('Projects').click()
```

Comment:
This link goes from disabled to enabled as the page loads. Updated the selector here to validate that it's enabled prior to clicking it.

Reply:
Hm. cy.click() waits for the `disabled` property to be false - https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Disability. So this could possibly be a bug with the button, where we have it `aria-disabled=true` but `disabled=false`?

Reply:
I'm not against merging this as-is, it's just a confusing state to be in because our actionability checks around clicking are pretty extensive.

Reply:
This is just a bog standard link with some attribute juggling to keep it accessible while still "disabled": https://github.com/cypress-io/cypress/blob/develop/packages/frontend-shared/src/gql-components/HeaderBarContent.vue#L58-L65. The actual `disabled` attribute isn't used here, so it might be sailing through our actionability checks?

Reply:
Hm, yeah. It looks like it's actually our actionability checks that are incomplete! We don't respect aria-disabled at all. 

I'm a little surprised there's no existing cypress issues that mention `aria-disabled`. Let's merge this as-is, and do you mind opening a follow-up to add aria-disabled to our actionability checks?

Reply:
Good call, logged: https://github.com/cypress-io/cypress/issues/22578